packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -124,16 +128,18 @@ export class MediaUpload extends Component {
 			id: URL_MEDIA_SOURCE,
 			value: URL_MEDIA_SOURCE,
 			label: __( 'Insert from URL' ),
-			types: [ MEDIA_TYPE_AUDIO ],
+			types: [ MEDIA_TYPE_AUDIO, MEDIA_TYPE_IMAGE ],
 			icon: globe,
 		};
 
+		// Only include `urlSource` option if `onSelectURL` prop is present
+		// (matches web behavior)
```

Comment:
Nitpick: I'm wondering if we could re-phrase it to remove the parenthesis block, wdyt?
```suggestion
		// Only include `urlSource` option if `onSelectURL` prop is present, in order to match the web behavior.
```